subsystems/visionsubsystem.py

```diff
@@ -121,6 +154,15 @@ def updateAdvantagescopePose(
         )
         SmartDashboard.putNumberArray(cameraKey, cameraPose)
 
+    def getNoteToCamera(self, note: PhotonTrackedTarget) -> Pose2d:
+        x = constants.kNoteCameraHeight / tan(
+            constants.kNoteCameraPitch - note.getPitch() * constants.kRadiansPerDegree
```

Comment:
whats the positive direction of pitch? if its positive pitches upwards, then kNoteCameraPitch would be inverted
if its positive pitches downwards that makes sense
for RHR it would be better as positive pitches upwards

Reply:
positive is downwards